Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -1489,4 +1489,11 @@
     <!-- XBUT: Abort button for test result positive no consent screen -->
     <string name="submission_test_result_positive_no_consent_button_abort">"Cancel"</string>
 
+    <!-- XHED: Title for the contact diary card displayed in the homescreen -->
+    <string name="contact_diary_homescreen_card_header" />
+    <!-- XTXT: Body for the contact diary card displayed in the homescreen  -->
+    <string name="contact_diary_homescreen_card_body" />
+    <!-- XBUT: Button text for the contact diary card button in the homescreen   -->
+    <string name="contact_diary_homescreen_card_button_text" />
+
 </resources>
```

Comment:
\n

Reply:
I thought we could just add the string names in here but empty. Haven't had complaints so far :D